src/view/dialogIngenValgt.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24"><g stroke="#000" stroke-linejoin="round" stroke-miterlimit="10" fill="none"><path d="M7.44 14.564c-.489-.066-.971-.17-1.44-.314l-5 2 2-3.5c-1.573-1.259-2.5-3.125-2.5-5.064 0-3.83 3.806-6.936 8.5-6.936s8.5 3.106 8.5 6.936"/><path d="M9.5 16.25c0 4.289 5 7.5 9.5 5.5l4 1.5-1.5-3c1.296-1.037 2-2.402 2-4 0-3.154-3.134-6-7-6s-7 2.846-7 6z"/></g></svg>
```

Comment:
er ikke dette samme svg icon-et som det forrige?

Helst gjennbruk ikoner. Ikke kopier dem. Vi kan ganske enkelt nå gå på en feil når vi ønsker å endre det ene iconet

Reply:
Avtalt som placeholder i påvente av to svg filer fra Linn